### PR TITLE
[TASK] Improve DatabaseRecords examples (#4809)

### PR DESCRIPTION
**Pull request by Alagts**
Resolves: #4809

Releases: main

### CODE REVIEW

File: Documentation/ApiOverview/DatabaseRecords/Index.rst [hunk 1]
```diff
@@ -35,15 +35,20 @@ Page records
     These represent pages in the page tree, which structure the website. They
     are stored in table :sql:`pages`.
 Content records
-    These include text, images, videos, or any other element placed within a
-    page. They are stored in table :sql:`tt_content`.
+    Every content record consists of some sub entities like texts, images,
```
Comment (sarahmccarthy123) [inline Documentation/ApiOverview/DatabaseRecords/Index.rst:38]:
```suggestion
    Every content record consists of sub entities like text, images,
```

File: Documentation/ApiOverview/DatabaseRecords/Index.rst [hunk 2]
```diff
@@ -35,15 +35,20 @@ Page records
...
+    videos, and so on. Content records can be placed on a page. They are stored
+    in table :sql:`tt_content`. TYPO3 has some pre configured content elements
+    like for example `Header Only`, `Regular Text Element`, `Text & Images`,
+    and `Images Only`.
 Backend user records
-    Information about users who have access to the TYPO3 backend. They
-    are stored in table :sql:`be_users`. The backend user groups they are
-    organized in are stored in table :sql:`be_groups`.
+    The user records consist of information about the users who have access to
+    the TYPO3 backend. They are stored in table :sql:`be_users`. The backend
```
Comment (sarahmccarthy123) [inline Documentation/ApiOverview/DatabaseRecords/Index.rst:45]:
```suggestion
    the TYPO3 backend. They are stored in table :sql:`be_users`. Users are
```

File: Documentation/ApiOverview/DatabaseRecords/Index.rst [hunk 3]
```diff
@@ -35,15 +35,20 @@ Page records
...
+    user groups they are organized in are stored in table :sql:`be_groups`.
```
Comment (sarahmccarthy123) [inline Documentation/ApiOverview/DatabaseRecords/Index.rst:46]:
```suggestion
    organized in user groups which are stored in table :sql:`be_groups`.
```

**Review by sarahmccarthy123 — CHANGES_REQUESTED** [review]
Just a few language changes :-)

**Comment by Alagts** [comment]
Thank you Sarah for your suggestions. ;-)

**Review by brotkrueml — APPROVED** [review]

**Pull request merged**